Makefile

```diff
@@ -62,4 +62,12 @@ tag:
 package:
 	php build/packager.php
 
+static: static-phpstan static-cs-fixer
+
+static-phpstan:
+	docker run --rm -it -e REQUIRE_DEV=true -v ${PWD}:/app -w /app oskarstark/phpstan-ga:0.12.23 analyze
```

Comment:
can we use 0.12.25?

Reply:
We should probably add a command to regenerate the baseline file too

Reply:
What do you think about somehow centralizing the image version numbers for both the makefile and the Github Action?

Reply:
What about delivering the PR first that reduces the baseline errors. Thereafter we can think of incrementing the PHPStan version.

Reply:
Did update it to `.27` here as well